__tests__/Itinerary.test.js

```diff
@@ -0,0 +1,32 @@
+const Itinerary =  require('../src/Itinerary.js');
+
+describe('Itinerary', () =>{
+    describe('with iterineraries', () =>{
+        let dover;
+        let calais;
+        let itinerary;
+        beforeEach(() => {
```

Comment:
Looking at the tests you are doing here, only the second one actually requires the code in the `beforeEach`, so this could be moved into that test and reduce the workload a bit